docs/src/transforms.md

```diff
@@ -46,9 +46,51 @@ Future transforms won't be added to the public API while in alpha. But in these
 | [RequestThrottling](#requestthrottling)               |❌           | Alpha                 |
 <!--| [DebugRandomDelay](#debugrandomdelay)                 | ❌          | Alpha                 |-->
 
+### CassandraSinkCluster
+
+This transform will send/receive cassandra messages to a node within a cassandra cluster.
+Messages will be routed according to the configured `data_center` and `rack`.
+
+The fact that shotover is routing to multiple destination nodes will be hidden from the client.
+This is achieved by rewriting `System.local` and `system.peers`/`system.peers_v2` query results.
+The `system.local` will make shotover appear to be its own node.
+While `system.peers`/`system.peers_v2` will be rewritten to list the configured shotover peers as the only over nodes in the cluster.
+
+```yaml
+- CassandraSinkCluster:
+    # contact points must be within the configured data_center and rack.
+    # If this is not followed, shotover will still function correctly but shotover will communicate with a
+    # node outside of the specified data_center and rack.
+    first_contact_points: ["172.16.1.2:9042", "172.16.1.3:9042"]
+
+    # Shotover will never route messages outside of this data_center
+    data_center: "dc1"
+    # Shotover will always prefer to route messages to this rack but may route outside of the rack when nodes in the rack are unreachable.
+    rack: "rack1"
+
```

Comment:
Is this missing the `host_id` field from #744?

Reply:
yep, good catch.
I've added it now.

Reply:
Just curious, since I'm fine with this approach. Does this need to be set by the user? Is there a reason we couldn't generate it ourselves?

Reply:
Its documented in the PR description: https://github.com/shotover/shotover-proxy/pull/744